features/providers/check_pending_employment.feature

```diff
@@ -138,7 +138,7 @@ Feature: Check pending employment
     When I click 'Save and continue'
     Then I should be on a page showing "Confirm the following"
 
-    When I check "I confirm the above is correct and that I'll obtain a signed declaration from my client."
+    When I select the first checkbox
```

Comment:
[ditto](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/4821/files#r1068345409)